sgx/urts/src/panic.rs

```diff
@@ -2,20 +2,22 @@
 
 // Mobilenode-side support for sgx_panic crate impl
 //
-// If the enclave panics, it will pass us a message, which we write to stderr
+// If the enclave panics, it will pass us a message, which we log as critical.
 // The enclave is expected to call rsgx_abort itself after this
 
+use mc_common::logger::global_log;
 use std::str;
 
 #[no_mangle]
 pub extern "C" fn report_panic_message(msg: *const u8, msg_len: usize) {
     let panic_msg_bytes = unsafe { std::slice::from_raw_parts(msg, msg_len) };
 
     match str::from_utf8(panic_msg_bytes) {
-        Ok(v) => eprintln!("Enclave panic:\n{}\n", v),
-        Err(e) => eprintln!(
+        Ok(v) => global_log::crit!("Enclave panic:\n{}\n", v),
```

Comment:
How does this work? global_log uses slog_scope, which I believe you say cannot be linked into an enclave?

Reply:
I see, yeah, I got confused.